### PR TITLE
API-1789: tls: add /etc/kubernetes/kubeconfig to known CA bundles

### PR DESCRIPTION
**Pull request by vrutkovs**
Spin-off from https://github.com/openshift/origin/pull/29074

### CODE REVIEW

**Review by sanchezl — APPROVED** [review]

File: pkg/cmd/update-tls-artifacts/generate-owners/tlsmetadatainterfaces/helpers.go [hunk 1]
```diff
@@ -29,7 +29,8 @@ var (
 	}
 	onDiskCABundles = certs.CABundleInfoByOnDiskLocation{
 		{Path: "/etc/kubernetes/ca.crt"}: {OwningJiraComponent: "Machine Config Operator"},
-		{Path: "/etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/trusted-ca-bundle/ca-bundle.crt"}:          {OwningJiraComponent: "kube-apiserver"},
+		{Path: "/etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/trusted-ca-bundle/ca-bundle.crt"}: {OwningJiraComponent: "kube-apiserver"},
+		{Path: "/etc/kubernetes/kubeconfig"}: {OwningJiraComponent: "kube-apiserver"},
```
Comment (deads2k) [inline pkg/cmd/update-tls-artifacts/generate-owners/tlsmetadatainterfaces/helpers.go:33]: you sure? I don't mind, but it's surprising to me.

Comment (vrutkovs) [reply pkg/cmd/update-tls-artifacts/generate-owners/tlsmetadatainterfaces/helpers.go:33]: main cert is from MCO though, lets start with kube-apiserver and maybe reassing later?

**Pull request merged**